lib/logging/PrettyStream.js

```diff
@@ -102,118 +102,108 @@ class PrettyStream extends Transform {
 
         logLevel = '\x1B[' + codes[0] + 'm' + logLevel + '\x1B[' + codes[1] + 'm';
 
-        // CASE: bunyan passes each plain string/integer as `msg` attribute (logging.info('Hey!'))
-        // CASE: bunyan extended this by figuring out a message in an error object (new Error('message'))
-        if (data.msg && !data.err) {
-            bodyPretty += data.msg;
-
-            output += format('[%s] %s %s\n',
+        if (data.req) {
+            output += format('[%s] %s "%s %s" %s %s\n',
                 time,
                 logLevel,
-                bodyPretty
+                data.req.method.toUpperCase(),
+                data.req.originalUrl,
+                statusCode(data.res.statusCode),
+                data.res.responseTime
```

Comment:
think it's worth putting before mentioned [safe getter](https://lodash.com/docs/4.17.11#get) around properties here, e.g:
```
_.get(data, 'req.originalUrl')
_.get(data, 'res.responseTime')
```
think that was the reason behind try/catch being here in the first place (specifically `data.res.responseTime` is very [Ghost specific field](https://github.com/TryGhost/Ghost/blob/fcd275f6c098684ad615b02d0714cf38e5cd8f44/core/server/web/shared/middlewares/log-request.js#L13))